CONTRIBUTING.md

```diff
@@ -41,29 +41,29 @@ dependency updates, unnoticeable refactoring, and so on.
 
 #### Title
 
-Describe your change with a short imperative title, e.g.
+Describe your change with a short imperative title, such as
 
 > Change small unnoticeable bits
 
 If your change resolves a ticket (see [above](#ticket)), please make sure you
 prefix your pull request title with `MBS-XXX: ` in order for our issue tracker
-to link your pull request to that ticket, e.g.
+to link your pull request to that ticket, such as
 
 > MBS-1234567: Change things relevant to users
 
-If it **partially resolves** a ticket, use parenthesis, e.g.
+If it **partially resolves** a ticket, use parenthesis, such as
```

Comment:
Maybe it's just me or my dialect of English, but when I read "such as," it puts more focus on the object rather than verb, which sounds weird in some cases.

For example, instead of "use e.g. x," I hear "parentheses e.g. x," which is odd because "x" is not an example of parentheses, it's an example of a particular usage which contains parentheses, if that makes sense. :)


It sounds better to me with "in":
```suggestion
If it **partially resolves** a ticket, use parentheses, such as in
```

However, there are at least some that sound better without "in," like the one above:

```
Describe your change with a short imperative title, such as
> Change small unnoticeable bits
```

This one sounds good because in "title such as x," "x" is indeed a title.

Reply:
That actually seems sensible to me, made a few amendments.